plugin/multiplex/index.js

```diff
@@ -44,7 +44,8 @@ app.get("/", function(req, res) {
 
 app.get("/token", function(req,res) {
 	var ts = new Date().getTime();
-	var rand = Math.floor(Math.random()*9999999);
+	var rand = new Uint32Array(1);
+	window.crypto.getRandomValues(array)
```

Comment:
probably a typo @albert-ziegler? I think you meant to put `rand` in that function call

Reply:
Indeed. Thanks for spotting, @lirantal!